app/src/main/java/com/techcourse/service/AppUserService.java

```diff
@@ -0,0 +1,45 @@
+package com.techcourse.service;
+
+import com.interface21.jdbc.core.TransactionTemplate;
+import com.techcourse.dao.UserDao;
+import com.techcourse.dao.UserHistoryDao;
+import com.techcourse.domain.User;
+import com.techcourse.domain.UserHistory;
+import java.util.NoSuchElementException;
+import javax.sql.DataSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
```suggestion
import com.techcourse.dao.UserDao;
import com.techcourse.dao.UserHistoryDao;
import com.techcourse.domain.User;
import com.techcourse.domain.UserHistory;
import java.util.NoSuchElementException;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
```

사용하지 않는 import문 정리하면 좋을 것 같아여~

Reply:
정리했습니다~!